config/xsl/loadreport/sections/custom-values.xsl

```diff
@@ -204,6 +204,46 @@
             </div>
         </div>
 
+        <div class="section" id="custom-data-logs-summary">
+            <xsl:call-template name="headline-custom-data-logs-summary"/>
+
+            <div class="content">
+                <xsl:call-template name="description-custom-data-logs-summary"/>
+                
+                <table class="c-tab-content table-autosort:0">
+                    <thead>
+                        <tr>
+                            <th>Scope</th>
+                            <th>Size</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                    <xsl:choose>
+                    <xsl:when test="count(customLogs/customLog) &gt; 0">
+                    <xsl:for-each select="customLogs/customLog">
+	                    <xsl:sort select="scope"/>
+	                    <tr>
+	                    	<td>
+	                    		<a>
+                            		<xsl:attribute name="href"><xsl:value-of select="path"/></xsl:attribute>
+                            		<xsl:value-of select="scope"/>
+                            	</a>
+                            </td>
+	                    	<td><xsl:value-of select="format-number(size, '#,##0.###')"></xsl:value-of> B</td>
```

Comment:
Might be more user-friendly to print the size in KB, MB, etc. We have a helper template for that: `format-bytes.xsl`. See `network-table.xsl` for info on how to use that helper template. In addition, please right-align the text.